playkit/src/main/java/com/kaltura/playkit/PKPreferredTrackSelectionMode.java

```diff
@@ -0,0 +1,7 @@
+package com.kaltura.playkit;
+
+public enum PKPreferredTrackSelectionMode {
```

Comment:
Move to inner class of PKTrackConfig, rename to Mode.

Reply:
done
Please review